fe/src/pages/user/Profile/Index/ProfileIndex.jsx

```diff
@@ -30,91 +37,85 @@ function ProfileIndex() {
+                  img={currentUser.avatar}
+                  customStyle={{ width: 200, height: 200 }}
                 />
-              </div>
-            </div>
-            <div className="mt-2 d-flex justify-content-center">
-              <p className="fw-bold">
-                {currentUser.first_name?.toUpperCase()}{" "}
-                {currentUser.last_name?.toUpperCase()}
-              </p>
-            </div>
+              </Card>
 
-            {currentUser.id !== parseInt(localStorage.getItem("user_id")) && (
-              <Fragment>
-                <div className="row mt-1 mb-4 w-75 mx-auto">
-                  <Followers />
-                </div>
+              <LayoutCenterChildren>
+                <p className="fw-bold">
+                  {currentUser.first_name?.toUpperCase()}{" "}
+                  {currentUser.last_name?.toUpperCase()}
+                </p>
+              </LayoutCenterChildren>
 
-                <div className="mb-2 d-flex justify-content-center">
-                  <hr className="w-50" />
-                </div>
-              </Fragment>
-            )}
+              {currentUser.id !== getUserId() && (
+                <Fragment>
+                  <div className="row mt-1 mb-4 w-75 mx-auto">
+                    <Followers />
+                  </div>
 
-            <div className="mt-2 d-flex justify-content-center">
-              {currentUser.id === parseInt(localStorage.getItem("user_id")) ? (
-                <EditUser user={currentUser} />
-              ) : (
-                isUser() &&
-                (!followBool ? (
-                  <div>
-                    <button
-                      className="btn btn-outline-danger"
-                      onClick={handleToggleFollow}
-                    >
-                      Unfollow
-                    </button>
+                  <div className="mb-2 d-flex justify-content-center">
+                    <hr className="w-50" />
                   </div>
+                </Fragment>
+              )}
+
+              <LayoutCenterChildren>
+                {currentUser.id === getUserId() ? (
+                  <EditUser user={currentUser} />
                 ) : (
-                  <div>
-                    <button
-                      className="btn btn-outline-primary"
-                      onClick={handleToggleFollow}
-                    >
-                      Follow
-                    </button>
+                  isUser() &&
+                  (!followBool ? (
+                    <Button
+                      text={"Unfollow"}
+                      style="btn btn-outline-danger"
+                      handler={handleToggleFollow}
+                    />
+                  ) : (
+                    <Button
+                      text={"Follow"}
+                      style="btn btn-outline-primary"
+                      handler={handleToggleFollow}
+                    />
+                  ))
+                )}
```

Comment:
- change `followBool` to user readable.
- avoid repetitive component in conditional statements. 

```suggestion
                {isCurrentUser ? (
                  <EditUser user={currentUser} />
		      isUser() &&
			 <Button
			     text={isFollowed ? "Follow" : "Unfollow"}
			     style={isFollowed ? "btn btn-outline-primary" : "btn btn-outline-danger"}
			     handler={handleToggleFollow}
			 />)
                 )}
```